qualang_tools/config/waveform_tools.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+# Definition of pulses follow Chen et al. PRL, 116, 020501 (2016)
+
+
+def drag_gaussian_pulse_waveforms(
+    amplitude, length, sigma, alpha, detune, delta, substracted
+):
+    """
+    Function that returns DRAG pulses that compensate for leakage and also for the AC stark shift
+    that the leakage creates to the qubit frequency.
+
+    Envelope: gaussian
+
+    :param amplitude: amplitude of voltage
+    :param length: pulse length
+    :param sigma: gaussian standard deviation
+    :param alpha: DRAG coefficient
+    :param detune: frequency shift to correct for AC start shift
+    :param delta: f_21 - f_10 - detune
```

Comment:
what is delta? why do we have both alpha and delta if their only usage is alpha/delta?

Reply:
delta is the difference in energy between 2-1 and 1-0 energy levels. If you are using detuning it also will be included. The units of delta are in MHz, and alpha is unitless and usually range between 0 and 1 as reported in the literature. If you were to define just one value alpha/delta then you would have something in the range of 0 to 1e-6